easy_phi_client.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import json
+import urllib
+import logging
+
+import settings
+
+
+class PythonClient(object):
```

Comment:
it is kind of odd to name class *PYTHON* client in .py file. Naming practices in web frameworks:

    from django.test import Client  # note that we already know that it is django test client from import
    # or
    from tornado.httpclient import asyncHTTPClient

I would suggest naming it RESTClient or just Client